ontology/coap.ttl

```diff
@@ -0,0 +1,165 @@
+@prefix : <https://www.w3.org/2019/wot/coap#> .
```

Comment:
Should the prefix be `cov:` for consistency?

```suggestion
@prefix cov: <https://www.w3.org/2019/wot/coap#> .
```

Reply:
I think this change would require adding `cov:` to all definitions defined in this document, so for simplicity (and to be consistent with the other ontologies, where the same approach is used) I think I would keep it as is.

Reply:
I think I'd add it, but that's up to you.